docs/ERRORS.md

```diff
@@ -1,30 +1,84 @@
-# Mapping
+# Errors
 
-This document contains some information about issues you may run into, or tips and tricks that may help you when working on mappings.
+This document will help you with common errors, how to handle errors, and known issues plus workarounds.
 
-## Best Practices
+## Error Squashing Methods
 
-### Indexing Speed
+_Note_: If you are the first to discover the error, please raise it to the team in `#development` on Discord.
 
-There are couple things you can do to significantly improve your indexing speed:
+It is inevitable that there will be errors in production subgraphs! This document describes how to be the first to catch the errors, and correct them efficiently.
 
-- Set a startblock (Use the deployment block of the contracts, [startblock app](https://startblock.vercel.app) may help).
-- Avoid call handlers and block handlers. Also depending on the Ethereum node ran by an indexer, call handlers and block handlers may or may not be supported (esp. on alt-EVM chains).
-- Limit the number of contract calls you perform. If you do need to perform contract calls, save the data, so you won't have to do repeated calls.
+### Catching Errors
 
-### Versions and Grafting
+There are two types of errors we are concerned about.
 
-- When fetching a protocol entity, it important that that the versions get updated at regular intervals such as at the time of fetching the protocol entity (Recommended). The 3 versions that should be updated are subgraph version, schema version, and methodology version. If the version is not updated, the subgraph will not be able to fetch the latest version of the protocol entity in cases where indexing is rebooted using the grafting feature.
+1. Fatal errors (the subgraph stops indexing)
+2. Data errors (something in the data is now incorrect, e.g. oracle breaks)
 
-## Common Issues
+### Where to look
+
+Fatal errors will be apparent, if you see an error message in [Protocol Metrics](https://messari.io/protocol-explorer/all-protocols) that looks something like the following:
+
+![Home page error](./images/errors/home-page-error.png)
+
+![Protocol page error](./images/errors/protocol-page-error.png)
+
+Data errors will be less obvious and more nuanced. It is possible that there is an error, but we decide not to fix it. For example, in Compound V2 we chose to not fix the ETH price when a proposal they passed broke it. This was what actually happened on chain.
```

Comment:
When this happens, should it be mentioned in the methodology? In case it should, let's add a mention to it here. 

> Whenever something like this occurs, it should be reflected in the methodology document for that protocol

or something